common-service/src/main/java/org/sunbird/util/CassandraStartUpUtil.java

```diff
@@ -0,0 +1,128 @@
+package org.sunbird.util;
+
+import com.typesafe.config.Config;
+import java.text.MessageFormat;
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.LoggerEnum;
+import org.sunbird.common.models.util.ProjectLogger;
+import org.sunbird.common.responsecode.ResponseCode;
+import org.sunbird.helper.CassandraConnectionManager;
+import org.sunbird.helper.CassandraConnectionMngrFactory;
+
+/**
+ * Utility class for cassandra startup
+ *
+ * @author arvind .
+ */
+public final class CassandraStartUpUtil {
+
+  private static final Config config = ConfigUtil.getConfig();
+
+  private CassandraStartUpUtil() {}
+
+  /**
+   * This method will check the cassandra data base connection. first it will try to established the
+   * data base connection from provided environment variable , if environment variable values are
+   * not set then connection will be established from property file.
+   */
+  public static void checkCassandraDbConnections(String keySpace) {
```

Comment:
Creates Cassandra DB connection(s) based on configured mode (embedded or standalone).

keySpace Keyspace (sunbird, sunbirdplugin) of DB for which connection needs to be established

Rename checkCassandraDbConnections to createCassandraConnection.